modules/VagrantController.py

```diff
@@ -94,7 +94,7 @@ def simulate(self, target, simulation_techniques, simulation_atomics):
         # get ip address from machine
         self.check_targets_running_vagrant(target, self.log)
         target_ip = self.get_ip_address_from_machine(target)
-        runner = ansible_runner.run(private_data_dir='../attack_range_local/',
+        runner = ansible_runner.run(private_data_dir='.',
```

Comment:
`os.path.join(os.path.dirname(__file__))`

Reply:
All of the path references in this file are relative to the repo directory, so I kept with the existing convention. Personally I don't see `__file__`-relative pathing as an improvement.